espnet/tts/pytorch_backend/tts.py

```diff
@@ -329,8 +329,11 @@ def train(args):
 
     # freeze modules, if specified
     if args.freeze_mods:
+        if hasattr(model, "module"):
+            prefixed_freeze_mods = ["module." + x for x in args.freeze_mods]
```

Comment:
We must handle the single gpu case as well.

Reply:
Hi, I reflected your comment
thx.